src/components/layout/SoftwareDownload.tsx

```diff
@@ -36,7 +38,13 @@ const SoftwareDownload = ({
 
   return (
     <>
-      <header className="max-w-7xl flex flex-row mx-auto px-4 pt-32 pb-16 lg:(pt-48 pb-26) gap-16">
+      <header className="max-w-7xl flex flex-row flex-wrap mx-auto px-4 pt-32 pb-16 lg:(pt-48 pb-26) gap-16">
+        {archived && (
+          <div className="text-center px-4 py-8 -mt-16 font-bold bg-yellow-400 dark:bg-yellow-900 shadow-md rounded w-full">
+            Waterfall has been archived! It is no longer maintained or
```

Comment:
This message would show if a different project became `archived`. That's no good.